pipeline-steps/terraform-apply.yaml

```diff
@@ -0,0 +1,19 @@
+parameters:
+  - name: subscription
+    type: string
+  - name: workingDirectory
+    type: string
+  - name: environment
+    type: string
+  - name: planLocation
+    type: string
+
+steps:
+
+  - task: TerraformCLI@0
+    displayName: Terraform apply
+    inputs:
+      workingDirectory: ${{parameters.workingDirectory}}
+      command: apply
+      environmentServiceName: ${{ parameters.subscription }}
+      commandOptions: -auto-approve "${{ parameters.planLocation }}"
```

Comment:
Can add the inputs false to the command
https://learn.hashicorp.com/tutorials/terraform/automate-terraform#terraform-init-input-false